app/views/layouts/_menu.html.erb

```diff
@@ -15,7 +15,7 @@
         </ul>
         </div>
         <div class="">
-        <buttom class="rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">Sign in</buttom>
+            <%= link_to "Sign in", new_user_session_path, class: "rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"%>
```

Comment:
```suggestion
            <% if user_signed_in? %>
               <%= link_to "Sign out", destroy_user_session_path, method: :delete, class: "rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600" %> 
            <% else %>
              <%= link_to "Sign in", new_user_session_path, class: "rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"%>
           <% end %>
```

Reply:
YES!

Reply:
Done. `link_to` doesn't work with `DELETE`.

Reply:
Ahh yes, I think you'd need to use `button_for` since links will only GET 🤦🏼